src/main/java/io/cdap/plugin/dlp/DLPTransformPluginConfig.java

```diff
@@ -62,6 +65,13 @@
   @Nullable
   protected String customTemplatePath;
 
+  @Name(DLP_LOCATION)
+  @Description("Resource location of DLP Service "
+      + "(for more info: https://cloud.google.com/dlp/docs/specifying-location)")
```

Comment:
style: for consistent code style, indentation should be 2 spaces (like for customTemplateEnabled)